package.json

```diff
@@ -14,7 +14,8 @@
     "aos": "^2.3.4",
     "chart.js": "^4.4.1",
     "material-icons": "^1.13.12",
-    "next": "^13.5.4",
+    "next": "^15.1.3",
+    "placement_tracker_v2": "file:",
     "primereact": "^9.6.2",
```

Comment:
`placement_tracker_v2`
remove if not needed.

Reply:
ok